package.json

```diff
@@ -7,7 +7,7 @@
     "node": ">=5.0.0"
   },
   "scripts": {
-    "test": "mocha ./test/test",
+    "test": "npm run lint && mocha ./test/test",
```

Comment:
unfortunately, this makes the tests a little slower.  the speed of the tests is a super benefit, so I am conflicted on this change.  I could make another run-script called check?  and then we can have the ci run that?